website/docs/users/case_studies.md

```diff
@@ -75,3 +63,19 @@ You can see a list community articles [here](blogs_videos_and_articles), but her
 > How to set up Pact tests with Angular & Jest
 
 <a href="https://medium.com/@dany.marques/how-to-set-up-pact-tests-with-angular-jest-ae157f272428"><img src="/img/logos/brand/die-mobiliar.png" alt="Die Mobiliar Logo" width="150" /></a>
+
+# Pactflow case studies
```

Comment:
You can only have one header on a page, so this doesn't show as an anchor/link on the right hand side.

We would have to create it as `## Pactflow case studies` and then add another hash to the sub titles `### Boost Insurance`

You may then want to create a Pact header `## Pact` and add the additional hash for those sub titles `### Gov.UK Pay` but that isn't mega necessary, as it is implied on the page/website